system/alternatives.py

```diff
@@ -123,10 +142,17 @@ def main():
                 if not link:
                     module.fail_json(msg="Needed to install the alternative, but unable to do so as we are missing the link")
 
-                module.run_command(
-                    [UPDATE_ALTERNATIVES, '--install', link, name, path, str(DEFAULT_LINK_PRIORITY)],
-                    check_rc=True
-                )
+                if need_to_force:
```

Comment:
Rather than splitting these commands up with a conditional, I think it's best to DRY and create a separate variable for the command to be run.

E.g.

```python
cmd =  [UPDATE_ALTERNATIVES, '--install', link, name, path, str(DEFAULT_LINK_PRIORITY)]
if need_to_force:
    cmd.append('--force')
module.run_command(cmd, check_rc=True)
```

Not sure off the top of my head if you can just append the `--force`, but if that doesn't work you can just do `cmd.insert(1, '--force')`.